docs/OracleDatastream-cdcSource.md

```diff
@@ -53,49 +53,49 @@ See [Configure your source Oracle database](https://cloud.google.com/datastream/
+**Private Connection Name**: Name of the private connection. The network admins of the Google Cloud Platform project should create a VPC peering between the database VPC and the Datastream VPC (see [Use private connectivity](https://cloud.google.com/datastream/docs/source-network-connectivity-options#privateconnectivity) for details). THis is the name of the VPC peering they created. Only applicable when you choose `Private connectivity (VPC peering)` as your `Connectivity Method`.
 
-**Host (in the <b>Database Location</b> section):** Hostname or IP address of your Oracle server to read from.
+**Host**(in the **Database Location** section): Hostname or IP address of your Oracle server to read from.
 
-**Port:** Port number to use to connect to your Oracle server.
+**Port**: Port number to use to connect to your Oracle server.
 
-**System Identifier (SID):** The system identifier(SID) of the oracle database you want to replicate data from.
+**System Identifier (SID)**: The system identifier(SID) of the oracle database you want to replicate data from.
 
-**Username:** Username to use to connect to your Oracle server. 
+**Username**: Username to use to connect to your Oracle server. 
 
-**Password:** Password to use to connect to your Oracle server.
+**Password**: Password to use to connect to your Oracle server.
 
-**Replicate Existing Data:** Whether to replicate existing data from the source database. When false, any existing data in the source tables will be ignored, and only changes that happened after the pipeline started will be replicated. By default, existing data will be replicated. 
+**Replicate Existing Data**: Whether to replicate existing data from the source database. When false, any existing data in the source tables will be ignored, and only changes that happened after the pipeline started will be replicated. By default, existing data will be replicated. 
 
-**Project:** The Google Cloud Platform project that has enabled the Datastream API. It will default to the value of the system property `GOOGLE_CLOUD_PROJECT` or `GCLOUD_PROJECT` for CDAP and Data Fusion Project for Data Fusion. 
+**Project**: The Google Cloud Platform project that has enabled the Datastream API. It will default to the value of the system property `GOOGLE_CLOUD_PROJECT` or `GCLOUD_PROJECT` for CDAP and Data Fusion Project for Data Fusion. 
 
-**Datastream Service Account Key:** The service account key for the service account that will be used as the identity to call the Datastream API. It will default to the content of the file referred by the system property `GOOGLE_APPLICATION_CREDENTIALS` for CDAP and Data Fusion Service Account for Data Fusion.
+**Datastream Service Account Key**: The service account key for the service account that will be used as the identity to call the Datastream API. It will default to the content of the file referred by the system property `GOOGLE_APPLICATION_CREDENTIALS` for CDAP and Data Fusion Service Account for Data Fusion.
```

Comment:
change "will default" to "defaults"
Change "Data Fusion Service Account for Data Fusion" to "Service Account for Cloud Data Fusion"